schematools/permissions.py

```diff
@@ -130,13 +146,22 @@ def create_acl_from_schema(engine, ams_schema, role, scope, dry_run):
                         field_scope_set, table_scope_set, field.name, table_name
                     )
                 )
-                if scope in field_scope_set:
+                grantees = (
+                    [scope_to_role(scope) for scope in field_scope_set]
+                    if role == "AUTO"
+                    else [role]
+                    if scope in field_scope_set
+                    else []
+                )
```

Comment:
One-liners zijn mooi, maar niet met nested `if`s.

```suggestion
                grantees = []
                if role == "AUTO":
                    grantees = [scope_to_role(scope) for scope in field_scope_set]
                elif scope in field_scope_set:
                    grantees = [role]
```

En deze code is in 3 verschillende plekken (wel met andere sets), dus misschien verstandig om een aparte functie te zijn.

Reply:
Die oneliner if is idd slecht te lezen. De formatting van Black maakt het er ook niet mooier op ;-)

Reply:
ipv op 3 plekken zit het nu nog op 2 plekken, dus laat ik dat verder zo, een aparte functie zal het niet duidelijker maken.